lib/ClangImporter/ImportType.cpp

```diff
@@ -837,6 +837,12 @@ namespace {
     SUGAR_TYPE(Attributed)
     SUGAR_TYPE(MacroQualified)
 
+    ImportResult VisitDecltypeType(const clang::DecltypeType *type) {
```

Comment:
This change is not needed. It's held over from earlier when I was trying to get SFINAE working.